composer.json

```
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "~8.3.0 | ~8.2.0 | ~8.1.0",
+        "php": "^8.1",
```

PHP does not follow semver, so minor version will need to be tested individually while patch versions are assumed to be non-breaking.

```suggestion
        "php": "~8.4.0 | ~8.3.0 | ~8.2.0 | ~8.1.0",
```